source/fundamentals/crud.txt

```diff
@@ -4,15 +4,12 @@
 CRUD Operations
 ===============
 
-.. default-domain:: mongodb
-
 .. toctree::
    :caption: CRUD Operations
 
    /fundamentals/crud/read-operations
    /fundamentals/crud/write-operations
    /fundamentals/crud/compound-operations
-   /fundamentals/crud/run-command
    /fundamentals/crud/write-read-pref
 
 CRUD (Create, Read, Update, Delete) operations enable you to work with
```

Comment:
Issue: Can't comment directly there, but remove line 26 from this file.
Instead, there should be a line in [this file](https://github.com/mongodb/docs-golang/blob/master/source/includes/fundamentals-sections.rst) that corresponds to the page